src/main/java/picard/vcf/GenotypeConcordance.java

```diff
@@ -546,19 +692,15 @@ final public static TruthAndCallStates determineState(final VariantContext truth
                 // Truth reference is shorter than call reference
                 final String suffix = getStringSuffix(callRef, truthRef, "Ref alleles mismatch between: " + truthContext + " and " + callContext);
                 truthRef = truthRef + suffix;
-                if (null == truthState) {
-                    truthAllele1 = truthGenotype.getAllele(0).getBaseString() + suffix;
-                    truthAllele2 = truthGenotype.getAllele(1).getBaseString() + suffix;
-                }
+                truthAllele1 = truthGenotype.getAllele(0).getBaseString() + suffix;
+                truthAllele2 = truthGenotype.getAllele(1).getBaseString() + suffix;
```

Comment:
Does this work properly on indels that are not repetitive?  I.e. shouldn't the `suffix` be spliced in between the shorter ref string and the rest of the allele?  My brain hurts thinking about this.